src/main/java/io/castle/client/model/Review.java

```diff
@@ -0,0 +1,26 @@
+package io.castle.client.model;
+
+public class Review {
```

Comment:
this needs to be updated?

Reply:
We have questions about the review endpoint:
GET castle.io /v1/reviews/{review_id} 

From where the user should get the review_id parameter??

Yes, the model needs to be fixed.

Reply:
It comes from a webhook